### ISSUE TITLE
Pluggable Talos Dashboard

### ISSUE THREAD

**Issue by salkin** [siderolabs/talos#9868]
## Feature Request
Support for user pluggable dashboard via extension service?

### Description
Make it possible for third-party dashboard as plugin to replace the existing Talos Dashboard. 

It should be possible to make fully configurable dashboard screens for third party use cases where configuration might be done by components running ontop of k8s, where direct machine config access is not desirable.

Optionally make Talos existing dashboard screens also pluggable into a third-party dashboard?

**Comment by smira** [comment]
I think this is already mostly there, probably some small bells & whistles are missing, but you can disable core dashboard and you could do your own dashboard as an extension service. 

Extension services run in a privileged context (as any extension service does), so it has full access to machined API (if desired). 

Not sure "about generic dasbhoard screens" done by components running on top of K8s (?). Not sure how that might even look like.

`talosctl dashboard` is fully open-source and you can plug it anywhere.

**Comment by salkin** [comment]
Yes this issue was more that we could get the small bells & whistles to be able to put the own created dashboard in-place of the core dashboard without having to patch Talos code itself

**Comment by frezbo** [comment]
I believe you can already add a new tab to the dashboard with just an extension

**Comment by frezbo** [comment]
From slack:

```text
I'm kind of able to do something by creating a service that mounts the /dev folder and then open and write in /dev/tty3
Then I just have to press Alt + F3 to show this but I feel it's a bit ugly.
```
